tensorflow/go/genop/generate.sh

```diff
@@ -24,6 +24,15 @@ then
   GOPATH=$(go env GOPATH)
 fi
 
+# Check if it is running in git-bash
+ls -la / | grep git-bash.exe
+if [ $? -e "0" ]
+then
+  GOPATH=${GOPATH//\\/\/}
```

Comment:
Couldn't this also be addressed by making the GOPATH env variable use UNIX-style paths?

Not sure this is robust enough to add to the code, since it's a bit hacky and will be hard for future maintainers to update.